enterprise/backend/src/metabase_enterprise/semantic_search/settings.clj

```diff
@@ -114,7 +114,7 @@
 (defsetting ee-search-indexer-exit-early-cold-duration
   "Number of seconds indexer should wait to see new data before yielding back to quartz."
   :type :integer
-  :default 30
+  :default 5
```

Comment:
30s is longer than the quartz interval for the indexer job, so we were just get rescheduled immediately after yielding
control from `indexing-loop`.

Reply:
oops, had to revert this. this value needs to be greater than the `dlq-frequency`. this does mean we get rescheduled immediately when we yield back to quartz, so possibly could tweak these later, but for now just revert.